modulefiles/module_base.wcoss_dell_p3

```diff
@@ -61,3 +45,13 @@ module load python/3.6.3
 
 # waveprep
 module load cdo/1.9.8
+
+# Workaround for github hpc-stack issues #370
+setenv LMOD_EXACT_MATCH no
```

Comment:
please set this variable immediately after `EnvVars` module is loaded.

Reply:
LMOD_EXACT_MATCH=no effectively turn off all version check. It is not recommended practice.
The reason of setting LMOD_EXACT_MATCH=no is to provide a temporary workaround to address a known bug in wcoss_dell_p3 hpc-stack. Therefore, LMOD_EXACT_MATCH=no should only be used in the module impacted by this bug.

Reply:
`LMOD_EXACT_MATCH=no`, does not turn OFF version check.  It enables loading a `default` version designated, if a version is not specified.  It has no impact if a module is already loaded.

WCOSS2 does not require this any more.

Reply:
Answered in Files changed page.

Reply:
@aerorahul 
This file module_base.wcoss_dell_p3 is not for wcoss2. 
Do you want me to set LMOD_EXACT_MATCH=no for the entire module_base.wcoss_dell_p3 file? 
Do you want me to not set LMOD_EXACT_MATCH=yes at the end of module_base.wcoss_dell_p3 file?

Reply:
In a global-workflow PR's meeting on 12/8 3:45p, Rahul requested to set LMOD_EXACT_MATCH=no for the entire module_base.wcoss_dell_p3.